lib/realtime/vehicle_or_ghost.ex

```diff
@@ -3,60 +3,112 @@ defmodule Realtime.VehicleOrGhost do
 
   @type t :: Vehicle.t() | Ghost.t()
 
+  @spec take_limited_matches([t()], Server.search_params()) :: %{
+          matching_vehicles: [t()],
+          has_more_matches: boolean()
+        }
+  def take_limited_matches(vehicles, %{text: text, property: search_property, limit: limit}) do
+    search_terms = text |> clean_for_matching |> Enum.reject(&(String.length(&1) < 2))
```

Comment:
**question:** this only applies to the new search flows, right? If so, shouldn't it be a minimum of 3 characters, not 2 (though the frontend will also enforce this limit in practice)?

Reply:
I think we want to keep the limit to 2 based on Mariele's comment [here](https://app.asana.com/0/0/1204650425301364/1205104040335462/f) - autocomplete has a minimum of 3 characters, but search still has a minimum of 2

Reply:
Ahh okay I neglected that consideration, but good call.